.github/renovate.json5

```diff
@@ -304,6 +304,17 @@
       "allowedVersions": "/^1\\.21\\.[0-9]+-?(alpine)?$/",
       "matchBaseBranches": [
         "v1.0",
+      ]
+    },
+    {
+      // update go version <1.22 for stable branches
```

Comment:
I'm reading this as <1.22 so <=1.21.x, but I guess the intention is to update to latest 1.22.x, correct?